packages/components/addon/components/hds/pagination/nav/index.js

```diff
@@ -75,6 +79,57 @@ export default class HdsPaginationNavIndexComponent extends Component {
     return this.currentPage === this.totalPages;
   }
 
+  elliptize({ pages, current }) {
+    const limit = 7; // limit # of page numbers shown at a time (should always be an even number!)
```

Comment:
@didoo Are you sure it shouldn't say it should be an "odd" number? (vs. "even") or should the default limit be changed?

Reply:
Also right now the limit isn't exposed as an argument, should it be?

Reply:
> @didoo Are you sure it shouldn't say it should be an "odd" number? (vs. "even") or should the default limit be changed?

Yes, my bad.

> Also right now the limit isn't exposed as an argument, should it be?

I though the same, yes, but wasn't sure :) As you prefer/think is better (in any case if we had to add it later as option, it would not be a breaking change).

Reply:
Ok, not sure if this feature (truncation) would be used enough to require it so I think we could add it later if requested.